### DISCUSSION TITLE
Batch replace multiple styles / actors at a time.

### DISCUSSION THREAD

**Discussion by oscdat** [SubtitleEdit/subtitleedit#6153] [Ideas]
Is it possible to implement a feature that can replace multiple styles / actors at once ( Batch Convert ). So for example if let's say there are 2 common styles / actors in all the selected subtitle files, then a way to

Replace the "Default" style in all the selected subtitle files with an user-set "Default1" style
and "Music" with "Music1" style

You get the idea. This would be really helpful, please consider. Thank you.

P.S. I think I'm doing something wrong here but the "Remove style/actor" under Batch Convert is not working for me. I've entered the exact style name with proper font case. Still I'm not able to delete it, let alone replace it.

**Comment by niksedk** [comment]
Oh, the existing "Remove style/actor" is actually "remove lines"... Updated the UI a little to clarify this:
![image](https://user-images.githubusercontent.com/1261331/185145806-08101614-1867-4211-90ba-9ef5d758be2a.png)

It's not possible to batch replace styles or actors atm,

**Reply by oscdat** [reply]
> It's not possible to batch replace styles or actors atm,

Thanks a lot for responding and sorry for the late reply, I thought my post went under the radar so I didn't check back. If it's not too much trouble to implement, I humbly request you to add this feature.
Thank you once again for your consideration and for this awesome software.
